src/components/containers/notification/Notification.js

```diff
@@ -11,9 +11,10 @@ import DismissButton from '../../controls/DismissButton';
 import { colors } from '../../theme';
 
 const DismissNotification = styled(DismissButton)`
-  line-height: 1.4;
+  line-height: 80%;
   opacity: 0.2;
-  padding: 6px 12px;
+  padding: 0;
+  padding-left: 15px;
```

Comment:
I'd probably use a margin-left here rather than padding.  When clicking on the X, the padding causes an irregular outline around the button element.

Reply:
Yep, I totally missed that. I've amended my commit to have margin-left instead of padding.